### ISSUE TITLE
bad reaction to typing 'ls'

### ISSUE THREAD

**Issue by Alexey-T** [veksha/cuda_example_terminal#3]
Xubuntu 20

[Screencast 2022-07-01 11:34:31.zip](https://github.com/veksha/cuda_example_terminal/files/9027424/Screencast.2022-07-01.11.34.31.zip)

**Comment by Alexey-T** [comment]
that is maybe a reason (not latest Cud)
```
Traceback (most recent call last):
  File "/home/user/.config/cudatext/py/cudatext.py", line 1165, in _timer_proc_callback_proxy
    return _live[info](tag)
  File "/home/user/.config/cudatext/py/cuda_example_terminal/__init__.py", line 230, in timer_update
    self.screen.memo_update()
  File "/home/user/.config/cudatext/py/cuda_example_terminal/MemoScreen.py", line 111, in memo_update
    self.apply_colors(x, y_memo, self.buffer[y_buffer])
  File "/home/user/.config/cudatext/py/cuda_example_terminal/MemoScreen.py", line 144, in apply_colors
    self.memo.attr(MARKERS_DELETE_BY_POS, x=x, y=y)
NameError: name 'MARKERS_DELETE_BY_POS' is not defined
ERROR: Exception in CudaText for cudatext._timer_proc_callback_proxy: NameError: name 'MARKERS_DELETE_BY_POS' is not defined
```

**Comment by veksha** [comment]
added `try .. except ..`
please, try again. does it fixes also issue #2 ?
